### PR TITLE
Counter Anti-RightClick: toptoon.com

### PR DESCRIPTION
**Pull request by ghost**
### URLs
https://toptoon.com/comic/ep_view/Sagak_romance/1
https://toptoon.com/comic/ep_view/lol/p1
https://toptoon.com/comic/ep_view/conveni/1

### Similarweb Web Traffic Report
https://www.similarweb.com/website/toptoon.com/#overview

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -41,6 +41,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||spush.toptoon.com/log/counter.jpeg?
```
Comment (zloyden) [inline SpywareFilter/sections/specific.txt:44]: I think you can add this `||spush.toptoon.com/log/`.

File: AnnoyancesFilter/sections/tweaks.txt [hunk 2]
```diff
@@ -1438,6 +1438,7 @@ lezhinus.com,lezhin.com###viewer-coach
 qanda.ai##div[class^="Picture__Protector-"]
 codedosa.com###wpcp-error-message
 !--- CSS ---
+toptoon.com#$#.document_img { pointer-events: auto!important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1441]: Please make an extra space after the property value in CSS rules.
```
toptoon.com#$#.document_img { pointer-events: auto !important; }
```

File: AnnoyancesFilter/sections/tweaks.txt [hunk 3]
```diff
@@ -1457,7 +1458,7 @@ codedosa.com#$#html { user-select: auto!important;-webkit-touch-callout: default
 open.spotify.com#$#span[dir="auto"][draggable="true"] { user-select: auto!important;-webkit-touch-callout: default!important;-webkit-user-select: auto!important;-moz-user-select: auto!important;-ms-user-select: auto!important; }
 kakaowebtoon.com,webtoon.kakao.com#$#img[class^="ViewerImage_image"] { pointer-events: auto!important; }
 lezhinus.com,lezhin.com#$#body * { pointer-events: auto!important; }
-appofmirror.com#$#body * { user-select: auto!important;-webkit-touch-callout: default!important;-webkit-user-select: auto!important;-moz-user-select: auto!important;-ms-user-select: auto!important; }
+toptoon.com,appofmirror.com#$#body * { user-select: auto!important;-webkit-touch-callout: default!important;-webkit-user-select: auto!important;-moz-user-select: auto!important;-ms-user-select: auto!important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/tweaks.txt:1461]: It doesn't work on my end. Check this
```
toptoon.com,appofmirror.com#$#html * { user-select: auto !important; -webkit-touch-callout: default !important; -webkit-user-select: auto !important; -moz-user-select: auto !important; -ms-user-select: auto !important; }
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 3 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: On Firefox Dev 95.0b2, my rule works.
But, It seems that your rule is better.
https://github.com/AdguardTeam/AdguardFilters/pull/98854/commits/05c8922fe869c4831b5026f5d456983e3618fa3e

File: AnnoyancesFilter/sections/tweaks.txt (hunk 2 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/tweaks.txt]: Done. https://github.com/AdguardTeam/AdguardFilters/pull/98854/commits/05c8922fe869c4831b5026f5d456983e3618fa3e

File: SpywareFilter/sections/specific.txt (hunk 1 above)
Comment (piquark6046) [reply SpywareFilter/sections/specific.txt]: Resolved. https://github.com/AdguardTeam/AdguardFilters/pull/98854/commits/e13b7915260332ec11794750da9fc7775c229b95

**Review by zloyden — APPROVED** [review]

**Pull request merged**